### PR TITLE
Adding design documentation for profile manager feature

### PR DESCRIPTION
**Pull request by gajjarvini**
1) Added a profile manager entity diagram
  2) Added profile manager overview diagram
  3) Added profile manager initialization sequence diagram

### CODE REVIEW

File: docs/images/profile_manager_entity.png.license [hunk 1]
```diff
@@ -0,0 +1,6 @@
+SPDX-FileCopyrightText: 2022  Great Software Laboratory Pvt. Ltd 
+
+SPDX-FileCopyrightText: 2022 Open Networking Foundation <info@opennetworking.org>
```
Comment (thakurajayL) [inline docs/images/profile_manager_entity.png.license:3]: just keep GSLAB name. Since this is new file added  by you

Comment (gajjarvini) [reply docs/images/profile_manager_entity.png.license]: I have made the changes

Comment (gab-arrobo) [reply docs/images/profile_manager_entity.png.license]: @thakurajayL, what are your thoughts on this PR? We're your comments addressed? Is this PR ready to merge?

**Review by gab-arrobo — APPROVED** [review]
+1

**Pull request merged**